### ISSUE TITLE
VSCodeInsider_Extension_VisualStudioOnline_ScreenReader:Voice over announces incorrect state  for the not selected controls "Details, Extension Pack, Contribution changelog" buttons.

### ISSUE THREAD

**Issue by vinodbk197** [microsoft/vscode#89091]
**Environment Details:​**
VSCode Version: 1.42.0-insider (System setup)
Operating System: MacOS
ScreenReader : VoiceOver
Mac version: 10.15.1 (19B88)

**Repro Steps:​**
1. Open VS Code insider 
2. Navigate till "Extension" button and search for the "Visual Studio Online"
3. Navigate  right panel till "Extension: Visual Studio Online" 
4. Start the VoiceOver.
5. Navigate to " Details, Extension Pack, Contribution changelog" buttons.
6. Observe screen reader announces. 

**Actual:**
Voice over announces incorrect state  for the not selected controls "Details, Extension Pack, Contribution changelog" buttons.

**Expected:​**
Voice over should  announces correct state  i.e selected/not selected  for the selected and not selected for the  controls "Details, Extension Pack, Contribution changelog" button

**​User Impact:​**
Screen reader user does not understand which control is selected/not selected if it does not announces.

**Comment by isidorn** [comment]
Duplicate of https://github.com/microsoft/vscode/issues/88792
